packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
```

Comment:
Nit: The Conventional Commit prefix you added to the PR title indicates that this is a fix, should this go under Fixed instead of Changed?

Reply:
Ow sorry @mcmire. I assumed the PR title should always use either fix or feat, but in this case, it’s more of a modification to the existing validation rather than a strict fix.

Reply:
Oh okay no worries then!